docs/_templates/partials/after_footer.njk

```diff
@@ -8,7 +8,7 @@
       elAdapter.parentElement.removeChild(elAdapter);
     }
   </script>
-  <script src="https://ab2b55b0a26bb32fdc01-2c8b94bcec8e01c366189541a4035ea4.ssl.cf5.rackcdn.com/custom-elements-es5-adapter.js"></script>
+  <script src="vendor/scripts/custom-elements-es5-adapter.js"></script>
```

Comment:
Can this be tracked by a package manager? I assume these checked-in scripts will be regenerated frequently without the aid of one.

Reply:
These scripts aren't regenerated, just copied. And the ES5 adapter is highly unlikely to change over time.

Reply:
I'm gonna see if 6bd77f7 will fix things by itself.